build_tools/python/benchmark_suites/iree/module_execution_configs.py

```diff
@@ -39,6 +39,12 @@ def _with_caching_allocator(
     loader=iree_definitions.RuntimeLoader.NONE,
     driver=iree_definitions.RuntimeDriver.CUDA)
 
+CUDA_CONFIG_LONG = _with_caching_allocator(
```

Comment:
It seems like we don't need to create a new execution config for long models. We can directly filter on the model's tags in `export_benchmark_config.py`?

Reply:
See comment below